docs/content/operations/quickstart-migration.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: doc_page
+title: "Migrating Derby Metadata and Local Deep Storage"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+# Migrating Derby Metadata and Local Deep Storage
+
+If you have been running an evaluation Druid cluster using the built-in Derby metadata storage and local
+deep storage (configurations used by the tutorial and single-machine quickstarts), and wish to migrate to a 
+more production-capable metadata store such as MySQL or PostgreSQL, and/or migrate your deep storage from local
+to S3 or HDFS, Druid provides the `export-metadata` tool to assist with such migrations.
+
+This tool exports the contents of the following Druid tables:
+- segments
+- rules
+- config
+- datasource
+- supervisors
```

Comment:
It looks that migrating only these tables would be enough for now, but if we extend this tool to support other types of deep storage in the future, then it would be probably worth to include `audit`, `tasks`, and `tasklogs`.

Reply:
agree, it would be useful to support those in the future